src/api/helpers/map-and-update-inactive-applications/index.ts

```diff
@@ -0,0 +1,30 @@
+import { Application, Context } from '.keystone/types'; // eslint-disable-line
+import mapAndGenerateInactiveApplicationsSaveArray from '../map-and-generate-inactive-applications-save-array';
+
+/**
+ * mapAndUpdateInactiveApplications
+ * maps and generates save object for inactive applications
+ * updates application statuses to abandoned
+ * @param {Array<Application>} applications: array of application ids and statuses
+ * @param {Context} context
+ */
+const mapAndUpdateInactiveApplications = async (applications: Array<Application>, context: Context) => {
+  try {
+    console.info('Mapping and updating inactive applications - mapAndUpdateInactiveApplications');
+
+    /**
+     * generates data array for database saving
+     * contains where and data (status change)
+     */
```

Comment:
Just thought - could you check the application's `updatedAt` fields are updated after calling `context.db.Application.updateMany`?

It might be doing so automatically; If not we should update it in the mapping function

Reply:
its done automatically